docs: update `Add SSL Certificate` article (#946)
devopness/devopness: docs/ssl-certificates/add-ssl-certificate.md

<details open>
  <summary>Steps to configure a DNS record pointing to your servers:</summary>

1. Copy the IP Address of the server where the application is running
    > Follow the {% mentionPost "/docs/servers/find-server-ip-address" %} guide for detailed instructions
1. Access your DNS service management panel
    > If you don't have your own domains configured with a DNS service, you can use free services such as [FreeDNS](https://freedns.afraid.org/) or hire paid managed DNS services such as [AWS Route53](https://aws.amazon.com/route53/), [Cloudflare DNS](https://www.cloudflare.com/dns/), or any [notable managed DNS service provider](https://en.wikipedia.org/wiki/List_of_managed_DNS_providers) that suits your needs.
1. Configure a DNS record for your application domain
    > Follow the specific instructions on your chosen DNS service documentation
1. Verify your DNS record is pointing to your Devopness managed server
  > This can be verified using a DNS propagation checker such as [whatsmydns](https://www.whatsmydns.net/) or [DNS Checker](https://dnschecker.org/)

</details>

> Once you have your application DNS record correctly pointing to the IP Address of your Devopness managed server, please follow the instructions below to issue a SSL Certificate to your application:

Reviewer: ```suggestion
Once you have your virtual host DNS record correctly pointing to the IP Address of your Devopness managed server, please follow the instructions below to issue a SSL Certificate to your virtual host:
```

> Once you have your application DNS record correctly pointing to the IP Address of your Devopness managed server, please follow the instructions below to issue a SSL Certificate to your application:

Reviewer: @CarolFP1896 the issue has not been prioritized or moved to any sprint, so between the merge of this PR and the PR that resolves that issue the users and Devopness team will no longer have any written resource to guide users that a DNS record should be created and how to do it.

Even with that task in Jira, I still believe that we should revert this change.

On the occasion that the task is being resolved, the PR can create the new article and delete these steps.

> Once you have your application DNS record correctly pointing to the IP Address of your Devopness managed server, please follow the instructions below to issue a SSL Certificate to your application:

Author: Ok, so let's revert it until the task is done, then we change it.

> 3. Click `View` in the `SSL Certificates` card to see a list of existing `SSL Certificates`

Author: Done!

> 5. On the Virtual host dropdown menu, choose a Virtual Host.

Reviewer 2: Whenever an UI element name is mentioned in our docs, please make sure to highlight it `like here` and use the exact upper case or lower case letters used for that element in the UI.
That way users can quickly relate to the UI elements when using our docs.

Please double check, cause I believe we spell `Virtual Host` capitalizing the first letters, but I might be mistaken.

Author: Done!

Reviewer 2: @CarolFP1896 
"choose a `Virtual Host` doesn't need to be high" is not referring to an UI element and therefore should not be highlighted.
I see you adopted this pattern to other names in the docs.
Please, as mentioned above, just highlight the "UI elements" and not all texts that mention the name of a resource type.

>    - The name of your SSL certificate will be the name of the chosen virtual host

Reviewer: ```suggestion
    - The name of your SSL certificate will be the name of the chosen virtual host
```

>    > The name of your SSL certificate will be the name of the chosen virtual host

Reviewer: @CarolFP1896 although they have the same visual effect, semantically, in Markdown, it is preferred to use unordered list notation; and in our docs, it is more common the unordered list approach [1].

```suggestion
   - The name of your SSL certificate will be the name of the chosen virtual host
```

[1]

```bash
$ grep -re '    > [A-Z]' `find docs -type f` | wc -l
44
$ grep -re '    - [A-Z]' `find docs -type f` | wc -l
85
```

Author: Done!

> 7. An action to install the SSL Certificate will start

Reviewer 2: Perhaps clarify who will start the action?

```suggestion
7. An action will be started by Devopness to issue an SSL Certificate and install it for the chosen Virtual Host
```

    related:
    featured:
required_permissions:
    - ssl-certificate:create
---

Before an SSL certificate can be issued, a valid DNS record must point to the server on which the virtual host is deployed, so the SSL Certificate can be issued and validated for the configured virtual host web domain. For example, if your virtual host is publicly accessible through the domain `my-application.example.com`, the subdomain `my-application` on the domain `example.com` must be configured to point to your server IP address.

<details open>
  <summary>Steps to configure a DNS record pointing to your servers:</summary>

1. Copy the IP Address of the server where the virtual host is running

>     - Follow the {% mentionPost "/docs/servers/find-server-ip-address" %} guide for detailed instructions

Reviewer: ```suggestion
    - Follow the guide {% mentionPost "/docs/servers/find-server-ip-address" %}
```
